GDJS/Runtime/runtimeobject.ts

```diff
@@ -712,6 +732,17 @@ namespace gdjs {
       return this.id;
     }
 
+    /**
+     * Get the network ID of the object.<br>
```

Comment:
No more  <br> in comments (it's from the past) + avoid vague "purpose" or starting sentences with "Or"

```suggestion
     * Get the network ID of the object.
     * 
     * The network ID is used to identify the object in a networked game or
     * in a save state.
```